scripts/docker/README

```diff
@@ -0,0 +1,37 @@
+Dockerfiles are used to create Docker images.  Docker images are the 'recipes' used to build virtual environments called containers.
```

Comment:
I have added to the README to discuss creating images and uploading to Docker Hub.
Regarding the failed tests.  It's a little confusing but it looks like the results you are seeing with the failures was produced by an automatically triggered test run that uses the merged pull request changes. You can see these results here:
https://dev.azure.com/axom/axom/_build/results?buildId=839&view=ms.vss-test-web.build-test-results-tab
Whereas the test results of the unmerged pull request are here:
https://dev.azure.com/healy22/Axom%20CI/_build/results?buildId=206&view=ms.vss-test-web.build-test-results-tab

Reply:
Thanks @keithhealy that's interesting -- one in is axom's azure space and the other is in your azure space.

I think we want to indicate failure  if either of these has test failures -- premerge or postmerge, since the latter will break tests for others.

Also, could you please merge develop into your branch (probably after fixing the test failure reporting), since I think it is out of date?